gio/src/subclass/action_group.rs

```diff
@@ -0,0 +1,371 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::ActionGroup;
+use glib::subclass::prelude::*;
+use glib::translate::*;
+use glib::{Cast, GString, ObjectExt, Quark, Variant, VariantType};
+use once_cell::sync::Lazy;
+use std::ptr;
+
+pub trait ActionGroupImpl: ObjectImpl {
+    fn action_added(&self, action_group: &Self::Type, action_name: &str);
```

Comment:
There are default implementations for many of these

Reply:
And this specifically is a signal default handler, so can stay `NULL` anyway

Reply:
Now it would simply panic instead of doing nothing. There is no default implementation of it, but because it's a signal handler nothing would happen if it stays `NULL`.

Reply:
I guess I have properly fixed it now :P